core/src/main/scala/better/files/ReaderInputStream.scala

```diff
@@ -51,7 +51,8 @@ class ReaderInputStream(reader: Reader, encoder: CharsetEncoder, bufferSize: Int
     encoderOut.flip()
   }
 
-  override def read(b: Array[Byte], off: Int, len: Int) = {
+  override def read(_b: Array[Byte] | Null, off: Int, len: Int) = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.